.github/workflows/cypress-demo-nx.yml

```diff
@@ -1,7 +1,7 @@
 name: Run e2e (Nx monorepo)
 on:
   pull_request:
-    types: [opened, labeled, unlabeled, synchronize]
+    types: [opened, synchronize]
```

Comment:
We're removing `labeled, unlabeled, ` to save build minutes I imagine? Also, it does seem off that we had those for as pull request types or was it part of some flow with release please?

Reply:
I think it was a mistake where something had been copy/pasted